### DISCUSSION TITLE
Command history/command recall feature for web terminal client

### DISCUSSION THREAD

**Discussion by ArmoredArtist** [GoMudEngine/GoMud#74] [Ideas]
I'd like to suggest adding a command history feature, where users can press the up/down arrow keys to cycle through their previously entered commands. I enjoy this feature to quickly repeat the same commands multiple times without retyping them.

**Comment by Volte6** [comment]
This exists in the telnet interface (if you connect via telnet). I'll add it to the web client as well... I agree, it's super useful.

**Comment by Volte6** [comment]
This has been added: https://github.com/Volte6/GoMud/pull/77

**Comment by Volte6** [comment]
Closing as completed
